ansible/roles/kong-api/defaults/main.yml

```diff
@@ -102,7 +103,22 @@ premium_consumer_rate_limits:
 #    - {name: 'rate-limiting', config.hour: "{{ medium_rate_limit_per_hour }}"}
 #    - {name: 'request-size-limiting', config.allowed_payload_size: "{{ medium_request_size_limit }}" }
 kong_apis:
-
+  - name: ItemSetAPIs
+    request_path: "{{ itemset_prefix }}/v1"
+    upstream_url: "{{ knowledge_mw_service_url }}/itemset/v1"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist: "{{ updateItemSet_ACL | default(['contentUpdate']) }}"
```

Comment:
add the variable as ItemSetAPIs_ACL instead of updateItemSet_ACL.

Reply:
@SMY-ALTAMASH - Done. Please check.